crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/validation/service/MeansAssessmentValidationService.java

```diff
@@ -32,11 +32,15 @@ public class MeansAssessmentValidationService {
     private final WebClient webClient;
     private final MaatApiConfiguration configuration;
 
+    String getUserIdFromRequest(MeansAssessmentRequestDTO meansAssessmentRequest) {
+        return meansAssessmentRequest.getUserSession().getUserName();
```

Comment:
Same comment as above, should we be checking if the UserSession is not null here?

Reply:
The userSession and the values it contains are all required fields in the schema, so any request that was missing those fields would be rejected with a 400.